pandas/core/computation/eval.py

```diff
@@ -89,7 +93,7 @@ def _check_resolvers(resolvers):
                                 'the __getitem__ method'.format(name=name))
 
 
-def _check_expression(expr):
+def _check_expression(expr: Any) -> None:
```

Comment:
Kind of vague from the docstring but looks like the type here should be `str` (Any should be avoided as it doesn't really offer much in the way of type checking)